### PR TITLE
Added image to sketch

### PR DESCRIPTION
**Pull request by sharur7**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #239 

### Type of change

**Select all that applies**:
* [x] New feature (non-breaking change which adds functionality)

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by sharur7** [comment]
@yashaswibiyahut @tejan-singh can you please review my PR?

File: Python/Image_Sketch/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Image to Sketch
+The original image is converted into black and white sketched image.
+
```
Comment (tejan-singh) [inline Python/Image_Sketch/README.md:25]: add requirements to run this script. Add command how to use this script. Please look for contribution guidelines before sending PRs. You need to write proper readme documentation.

**Review by tejan-singh — CHANGES_REQUESTED** [review]

File: Python/Image_Sketch/README.md (hunk 1 above)
Comment (sharur7) [reply Python/Image_Sketch/README.md:25]: I have updated readme.md with the appropriate changes. Can you please review it again? @tejan-singh

File: Python/Image_Sketch/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Install the following libraries through command line:
+```
+pip install opencv-python
+pip install numpy
+pip install scikit-image
+```
+### How to use it :
+1. Download or clone the repository
+2. Install required libraries
+3. Add path of your image in ```img = io.imread("enter your image path") ``` function from **Sketch.py**
```
Comment (tejan-singh) [inline Python/Image_Sketch/README.md]: you need to add functionality to add image path through command line

**Review by tejan-singh — CHANGES_REQUESTED** [review]

File: Python/Image_Sketch/README.md (hunk 2 above)
Comment (sharur7) [reply Python/Image_Sketch/README.md]: @tejan-singh added the functionality of the user input filename through command line

**Pull request merged**